model/src/main/kotlin/OrtResult.kt

```diff
@@ -175,7 +177,11 @@ data class OrtResult(
         val project = getProject(id)
 
         scanResultsById[id].orEmpty().flatMap {
-            FindingsMatcher().match(it.summary.licenseFindings, it.summary.copyrightFindings)
+            FindingCurationMatcher()
+                .applyAll(it.summary.licenseFindings, getLicenseFindingsCurations(id))
+                .let { findings -> FindingsMatcher().match(findings, it.summary.copyrightFindings)
+                .toSortedSet()
+            }
```

Comment:
Formatting is broken here. Should be 

```kotlin
.let { findings ->
    FindingsMatcher().match(findings, it.summary.copyrightFindings).toSortedSet()
}
```
or single line.

Reply:
Done.